main.go

```diff
@@ -12,13 +12,21 @@ import (
 
 func makeHandler(name string, fn func(http.ResponseWriter, *http.Request, string, string)) http.HandlerFunc {
```

Comment:
If you go to the webpage http://goreportcard.com/ it seems to only allow use github repos.  

My understanding after playing with this codebase is the `makeHandler` function creates all the handlers for other routes including `/badge` and `/report`, but those need a github username and repo as well.  Im hoping the owners of this repo can shed light on if this change breaks anything, but so far in my local testing it looks good. 

<img width="775" alt="screen shot 2015-08-23 at 5 47 19 pm" src="https://cloud.githubusercontent.com/assets/5687406/9430526/22ce4c32-49bf-11e5-8152-ddd0e1fe7478.png">